content/en/docs/components/katib/early-stopping.md

```diff
@@ -0,0 +1,204 @@
++++
+title = "Using Early Stopping"
+description = "How to use an early stopping in Katib experiments"
+weight = 60
+                    
++++
+
+This guide shows how you can use
+[early stopping](https://en.wikipedia.org/wiki/Early_stopping) to improve your
+Katib experiments. Early stopping allows you to avoid overfitting when you
+train your model during Katib experiments. It helps you to save computing
+resources and experiment execution time by stopping the experiment's trials
+before the training process is complete.
```

Comment:
AFAIK, early stopping helps with resources and execution when the (validation) loss or some other target metric no longer improves. Let's add that here to accommodate for the users who are new to ML or aren't as proficient in ML as the others.

For example:

```suggestion
Katib experiments. Early stopping allows you to avoid overfitting when you
train your model during Katib experiments. It also helps by saving computing
resources and reducing experiment execution time by stopping the experiment's trials
when the target metric(s) no longer improves before the training process is complete. 
```

Notice the use of "the target metric(s)"

Reply:
Agree, nice explanation!